tests/test_annotation.yml

```diff
@@ -5,20 +5,44 @@
     - snpeff
   files:
     - path: results/annotation/test/test_snpEff.ann.vcf.gz
+      md5sum: 01f24fdd76f73eefd695beea7b3d3d8e
     - path: results/annotation/test/test_snpEff.ann.vcf.gz.tbi
+      md5sum: 51e418d9be9bb33f1d4123493b15b6c9
     - path: results/multiqc
     - path: results/reports/snpeff/test/snpEff_summary.html
+      # snpEff_summary.html changes md5sums on reruns.
+      contains: ["<b> Genome total length </b>", "<td> 100,286,402 </td>", "<td> MT192765.1 </td>"]
     - path: results/reports/snpeff/test/test_snpEff.csv
+      # test_snpEff.csv changes md5sums on reruns.
+      contains:
+        [
+          "Values , 50,100",
+          "Count , 1,8",
+          "Reference , 0",
+          "Het , 1",
+          "Hom , 8",
+          "Missing , 0",
+          "MT192765.1, Position,0,1",
+          "MT192765.1,Count,0,0",
+        ]
     - path: results/reports/snpeff/test/test_snpEff.genes.txt
+      md5sum: 130536bf0237d7f3f746d32aaa32840a
 - name: Run VEP
   command: nextflow run main.nf -profile test,annotation --tools vep --skip_tools multiqc
   tags:
     - annotation
     - vep
   files:
     - path: results/annotation/test/test_VEP.ann.vcf.gz
+    # binary changes md5sums on reruns.
     - path: results/annotation/test/test_VEP.ann.vcf.gz.tbi
+      md5sum: 4cb176febbc8c26d717a6c6e67b9c905
     - path: results/reports/EnsemblVEP/test/test_VEP.summary.html
+      # test_VEP.summary.html changes md5sums on reruns.
+      contains:
+        [
+          '<h3 id="gen_stats">General statistics</h3><table class="stats_table"><tr><td>Lines of input read</td><td>0</td></tr><tr><td>Variants processed</td><td>0</td></tr><tr><td>Variants filtered out</td><td>0</td></tr><tr><td>Novel / existing variants</td><td>-</td></tr><tr><td>Overlapped genes</td><td>0</td></tr><tr><td>Overlapped transcripts</td><td>0</td></tr><tr><td>Overlapped regulatory features</td><td>0</td></tr></table>',
```

Comment:
Maybe would be better to have something with numbers in the contains? This sounds like some static test that it would output regardless of anything